### PR TITLE
Created a mobile version of a portfolio

### PR DESCRIPTION
**Pull request by vangelif**
I created a mobile version of a portfolio, showcasing the following:

1. cards for the recent projects
2. compartment to obtain CV
3. overview of the languages, frameworks and skills
4. an introduction and a contact message to connect

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
...
+          I 'm always<br />
+          interested in<br />
+          hearing about<br />
+          new projects, so<br />
+          if you'd like to<br />
+          chat, please get<br />
+          in touch.
+        </h1>
+      </div>
     </section>
```
Comment (IngDavidTM) [inline index.html]: - [x] Good job creating all the required elements inside your project :+1:, but I noticed that your HTML Structure is not using the right HTML semantics. The section element is a generic standalone section of the document, so use it to group elements that are related together. There are two main sections in the project that you should create, the works section and the about myself section, so please separate them :pray:. To know more about `<section>` tag [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: index.html [hunk 2]
```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
...
+          <img src="images/scroll-button.png" alt="scroll-button" />
+        </a>
+      </div>
+    </section>
+    <!-- *start of work section -->
+    <section class="works">
+      <!-- h1, a line, 6 x cards, 
+        resume, languages, frameworks, skills  -->
+      <div>
+        <h1>My Recent Works</h1>
```
Comment (IngDavidTM) [inline index.html]: - [x] Good job identifying the titles and subtitles :+1:, but the title (e.g., "My Recent Works") doesn't have to use `<h1>` tag. You could use `<h2>` tag. This is essentially due to SEO reasons which many believe it is better to have only one `<h1>` tag on a webpage.

File: index.html [hunk 3]
```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
...
+          <a href="#"
+            ><img class="button3" src="images/jvscr.png" alt="javascr-button"
+          /></a>
+          <a href="#"
+            ><img class="button4" src="images/html.png" alt="html-button"
+          /></a>
+        </div>
+        <div class="project-button">
+          <a href="#"><img src="images/greenbtn.png" alt="go-to-project" /></a>
+        </div>
```
Comment (IngDavidTM) [inline index.html]: - [x] As a project requirement you shouldn't use images (`` tag)for the button and elements, instead, you should use elements to create the structure of your page. 

- [x] Also, we need to follow the button styling: `Active`, `Hover`, and `Pressed`, Please add these effects for a better user interface.

Screenshot from 2023-01-14 13-09-34

File: index.html [hunk 4]
```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
...
+          <a href="#"
+            ><img class="button2" src="images/css.png" alt="css-button"
+          /></a>
+          <a href="#"
+            ><img class="button3" src="images/jvscr.png" alt="javascr-button"
+          /></a>
+          <a href="#"
+            ><img class="button4" src="images/html.png" alt="html-button"
+          /></a>
+        </div>
```
Comment (IngDavidTM) [inline index.html]: - [x] Good job setting all the elements here :+1: but as the project requires it should be a list, so you should use `<li>` tags for each `<a>` element and wrap all of them using `<ul>` :+1:

File: index.html [hunk 5]
```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
...
+            ><img class="cpen-btn" src="images/skills/Codepen.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="gitl-btn" src="images/skills/Gitlab.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="term-btn" src="images/skills/Terminal.png" alt=""
+          /></a>
+        </div>
+      </div>
```
Comment (IngDavidTM) [inline index.html]: - [x] Your design here is awesome :100:, but it is required to make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul` with each being inside `li`, And also you need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` :+1:

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @VangeliVangeli  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -4,76 +4,453 @@
...
+                  ><button class="btn-hover btn-active btn-pressed">
+                    HTML
+                  </button></a
+                >
+              </li>
+            </ul>
+            <button class="section-button btn-hover btn-active btn-pressed">
+              See Project
+            </button>
+          </div>
```
Comment (algerina) [inline index.html:120]: - [Optional ] You did well implementing the changes required by the previous reviewer for the button styling: Active, Hover, and Pressed. Consider using the same colors as the Figma design.

**Review by algerina — APPROVED** [review]
Hi @VangeliVangeli ,
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
You did great implementing the previews TSE's requirements :+1: 
Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**